### DISCUSSION TITLE
Adding Search field or increase pagination count to "My Sites" page.

### DISCUSSION THREAD

**Discussion by rewebit** [plausible/analytics#4247] [Self-Hosted Support]
Hello,

Just wanted to ask if anyone had implemented a search field or tried to increase the page count for the number of domains displayed under the "My Sites" page? we have a fair number of partners who are running under our self hosted instance.

It would be nice to have to go through fewer pages or even better search on domain name, to quickly find their website.

thank you for any help that can be sent along. Have a great day!

**Comment by ruslandoga** [comment]
👋 @rewebit 

v2.1 should have site search available.

**Reply by ruslandoga** [reply]
https://github.com/plausible/analytics/discussions/4125

**Reply by rewebit** [reply]
...yes... Thank you for the quick response! That would have been the smart thing to do...
